test/unit-test-helpers/mock_ima.js

```diff
@@ -15,6 +15,8 @@ google =
     },
     Ad : function()
     {   //see https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.Ad
+
+      this.g = { vpaid : false };
```

Comment:
What is this.g?

Reply:
It's a container in the Ad object returned by the sdk... Google does not provide a getter function for the vpaid property stored there for some reason, so I had to grab it from the object directly. See line 1711 in google_ima.js. I realize it is hackish, but it seemed the best way to tell if the ad was vpaid or not

Reply:
That's fine, works for me!